### DISCUSSION TITLE
Meeting 24/04/2024

### DISCUSSION THREAD

**Discussion by Ivanderkelen** [CORDEX-be2/ValEnsPy#28] [Meeting minutes]
### Decisions
#30 Discussion points

**Comment by kobebryant432** [comment]
Biggest question to answer next meeting:
- Preprocessing (Use CDO): Use CDO or not? - In particular for complex grids. 
- MAR pre-processor (@nicghil) #4 
- CF convention checks (What to check)? #9 and #7 

Intern: Focus on the water cycle verification - start in May 21st.
ERA-5 (@Ivanderkelen) & ERA-5 Land.
4 CORDEX.be I one models - one grid and data format! (Evaluation run of MAR).
